packages/pyright-internal/src/languageServerBase.ts

```diff
@@ -489,6 +489,8 @@ export abstract class LanguageServerBase implements LanguageServerInterface, Dis
 
     protected getSeverityOverrides(value: string | boolean): DiagnosticSeverityOverrides | undefined {
         const enumValue = parseDiagLevel(value);
+        // TODO: this allows additional diagnostic severity levels (unreachable, unused & deprecated) on all rules
```

Comment:
issue?

Reply:
i think its covered by #64

the config in pyright seems to be the messiest part of the codebase. if i ever get around to rewriting it all of it this will be deleted anyway